web/src/services/api.ts

```diff
@@ -265,7 +265,7 @@ export default class API {
     });
   }
 
-  subscribeToNewsletter(email: string): Promise<void> {
+  subscribeToNewsletter(email: string): Promise<{ token: string }> {
```

Comment:
should be void again, since we don't return the token anymore, right? :)

```suggestion
  subscribeToNewsletter(email: string): Promise<void> {
```